extra_docs/sensitivities/variance_sensitivity.tex

```diff
@@ -0,0 +1,133 @@
+%
+%\begin{align*}
+%\Delta \sigma^2 &\le 2M \left(\frac{M-m}{n}\right) - \left(\frac{M-m}{n}\right)^2 +\frac{M-m}{n-1}.
+%\end{align*}
+%\end{proof}
+%
+%\begin{lemma}
+%If $\forall x_i \in \x, x_i \in (0,1),$
+%$$ \Delta \sigma^2 \le \frac{1}{n-1} + \frac{2}{n} - \frac{1}{n^2}.$$
+%\end{lemma}
+%\begin{proof}
+%This follows directly from the above theorem, setting $m=0$ and $M=1$. 
+%\end{proof}
+
+\begin{lemma}
+\label{lemma:meansum}
+For arbitrary $a$,
+$$ \sum_{i=1}^n (x_i - a)^2 = \sum_{i=1}^n (x_i - \bar{x})^2 + n(a-\bar{x})^2.$$
+\end{lemma}
+
+\begin{proof}
+\begin{align*}
+\sum_{i=1}^n (x_i - a)^2 &= \sum_{i=1}^n \left( (x_i - \bar{x}) - (a-\bar{x}) \right)^2\\
+	&= \sum_{i=1}^n \left( (x_i - \bar{x})^2 -2(x_i - \bar{x})(a-\bar{x}) + (a-\bar{x})^2\right)\\
+	&= \sum_{i=1}^n (x_i - \bar{x})^2 - 2\sum_{i=1}^n \left(x_ia-x_i\bar{x} -\bar{x}a + \bar{x}^2\right) + \sum_{i=1}^n \left( a^2 -2a\bar{x} + \bar{x}^2\right)\\
+	&=  \sum_{i=1}^n (x_i - \bar{x})^2 -2a\sum_{i=1}^n x_i + 2\bar{x}\sum_{i=1}^n x_i + 2\bar{x}an - 2\bar{x}^2n + a^2n-2a\bar{x}n+\bar{x}^2n\\
+	&=  \sum_{i=1}^n (x_i - \bar{x})^2 + a^2n-2a\bar{x}n+\bar{x}^2n\\
+	&=  \sum_{i=1}^n (x_i - \bar{x})^2 + n(a-\bar{x})^2
+\end{align*}
+
+\end{proof}
+
+\begin{theorem}
+Let 
+$$ f(\x) = \sum_{i=1}^n (x_i - \bar{x})^2.$$
+Then for $\x$ bounded between $m$ and $M,$ $f$ has sensitivity bounded above by
+$$\frac{n-1}{n} (M-m)^2.$$
+\end{theorem}
+
+\begin{proof}
+Consider database $\x$ with $n$ points and database $\xprime = \x \cup \{x_{n+1}\}.$ Note that
+
+\begin{align}
+\label{eqn:meanprime}
+\bar{x}' &= \frac{1}{n+1} \sum_{i=1}^{n+1} x_i \nonumber \\
+	&= \frac{n\bar{x} + x_{n+1}}{n+1}.
+\end{align}
+
+Then,
+\begin{align*}
+f(\xprime) &= \sum_{i=1}^n (x_i - \bar{x}')^2 + (x_{n+1} - \bar{x}')^2\\
+	&= \sum_{i=1}^n (x_i - \bar{x})^2 + n(\bar{x}'-\bar{x})^2 + (x_{n+1} - \bar{x}')^2 &&\text{(By Lemma \ref{lemma:meansum})}\\
+	&= f(\x) + n\left( \frac{n\bar{x} + x_{n+1}}{n+1}-\bar{x}\right)^2 + \left(x_{n+1} - \frac{n\bar{x} + x_{n+1}}{n+1} \right)^2 &&\text{(By Equation \ref{eqn:meanprime})}\\
+	&= f(\x) + n\left(\frac{x_{n+1}-\bar{x}}{n+1}\right)^2 + \left( \frac{nx_{n+1}-n\bar{x}}{n+1}\right)^2\\
+	&= f(\x) + (x_{n+1}-\bar{x})^2 \frac{n+n^2}{(n+1)^2}\\
+	&= f(\x) + (x_{n+1}-\bar{x})^2 \frac{n}{n+1}\\
+	&\le f(\x) + (M-m)^2 \frac{n}{n+1}
+\end{align*}
+
+Consider database $\xprimeprime$, which is the same size as $\xprime$ but with a single point changed. Note that
+
+$$ \lvert f(\xprimeprime) - f(\xprime) \rvert \le f(x')-f(x),$$
+
+so the sensitivity of $f$ is bounded by how much it changes from $\x$ to $\xprime$, and thus the sensitivity of $f$ is bounded by 
+$$ (M-m)^2 \frac{n}{n+1}.$$
+
+Since we'd really like to consider the sensitivity of database $\xprime$, redefine $n$ as $n+1$ to give a sensitivity bounded above by 
```

Comment:
I am a bit confused by the argument here, though I think the answer is correct. I got to this same bound by considering the sensitivity if |x'| were n-1 rather than n+1.

Reply:
I think you might be right that we also have to consider it in that direction. Or at least it's safest to assume we do? I'll write up the proof in that direction as well.